collect_app/src/androidTest/java/org/odk/collect/android/feature/instancemanagement/SendFinalizedFormTest.kt

```diff
@@ -134,4 +140,40 @@ class SendFinalizedFormTest {
             .clickOK(SendFinalizedFormPage())
             .assertQuantityText(org.odk.collect.strings.R.plurals.forms_ready_to_send, 1, 1)
     }
+
+    @Test
+    fun formsAreSentInOldestFirstOrder() {
```

Comment:
I think it's worth explicitly setting the sort to "Date, newest first" here just to make sure the order isn't tied to the UI (which it was before).

Reply:
Done.